src/tempo.py

```diff
@@ -207,7 +207,9 @@ def _build_storage_config(self, s3_config: dict):
                 endpoint=s3_config["endpoint"],
                 secret_key=s3_config["secret-key"],
             ),
-            block=tempo_config.Block(version="v2"),
+            # starting from Tempo 2.4, we need to use at least parquet v3 to have search capabilities (Grafana support)
+            # https://grafana.com/docs/tempo/latest/release-notes/v2-4/#vparquet3-is-now-the-default-block-format
+            block=tempo_config.Block(version="vParquet3"),
```

Comment:
Tempo will now give a warning that property `v2_out_buffer_bytes` in line `255` can be used when version is set to "v2". Should we worry about this?

Reply:
Removed the property altogether.